terratest/src/test/pega/deployment_utility.go

```diff
@@ -118,6 +118,9 @@ func VerifyDeployment(t *testing.T, pod *k8score.PodSpec, expectedSpec pegaDeplo
 	var envIndex int32 = 0
 	require.Equal(t, pod.Containers[0].Env[envIndex].Name, "NODE_TYPE")
 	require.Equal(t, expectedSpec.nodeType, pod.Containers[0].Env[envIndex].Value)
+	envIndex++
+	require.Equal(t, pod.Containers[0].Env[envIndex].Name, "PEGA_APP_CONTEXT_PATH")
+	require.Equal(t, pod.Containers[0].Env[envIndex].Value, "prweb")
```

Comment:
Does it make sense to add tests for values other than prweb?

Reply:
Sure.  I will add a test for another value.